### PR TITLE
Annex Sentry

### PR DESCRIPTION
**Pull request by AhmadYProjects**
[[Annex Sentry]]

### CODE REVIEW

**Comment by AhmadYProjects** [comment]
[[Annex Sentry]]

File: Mage.Sets/src/mage/cards/a/AnnexSentry.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+import java.util.UUID;
+
+/**
+ *
+ * @author AhmadYProjects
+ */
+public final class AnnexSentry extends CardImpl {
+
+    private static final FilterOpponentsArtifactCreaturePermanent filter = new FilterOpponentsArtifactCreaturePermanent("creature with cmc 3 or less");
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/a/AnnexSentry.java:27]: you don't need to create a whole new class for this, just add the necessary predicates to a filter that already exists, also it needs to be artifact or creature and not artifact creature

**Review by theelk801 — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/a/AnnexSentry.java (hunk 1 above)
Comment (AhmadYProjects) [reply Mage.Sets/src/mage/cards/a/AnnexSentry.java:27]: so something like this?

private static final FilterPermanent filter = new FilterPermanent("Artifact or creature an opponent controls with cmc 3 or less");

    static {
        filter.add(new ManaValuePredicate(ComparisonType.FEWER_THAN,4));
        filter.add(Predicates.or
                (CardType.ARTIFACT.getPredicate(),
                CardType.CREATURE.getPredicate()));
        filter.add(TargetController.OPPONENT.getControllerPredicate());
    }

Comment (theelk801) [reply Mage.Sets/src/mage/cards/a/AnnexSentry.java:27]: yeah, except the filter message shouldn't be capitalized

**Comment by theelk801** [comment]
don't forget to delete the class you added

**Comment by AhmadYProjects** [comment]
just did that

**Review by theelk801 — APPROVED** [review]
great, thanks for the contribution

**Pull request merged**